CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.10.0 (2020-03-19)
+
+- Added CSS classes for dark, light, and row borders. [#100](https://github.com/blackbaud/skyux-theme/pull/100)
```

Comment:
Should the classes have tick marks around them, or are they fine as regular text?

Reply:
I don't think so, since those aren't the exact CSS classes.

Reply:
Cool, just wanted to confirm.